src/service/resources-impl.js

```diff
@@ -237,19 +237,15 @@ export class ResourcesImpl {
       const root = /** @type {?Element} */ (this.ampdoc.isSingleDoc() && iframed
         ? /** @type {*} */ (this.win.document)
         : null);
-      try {
-        this.intersectionObserver_ = new IntersectionObserver(
-          (e) => this.intersect(e),
-          // rootMargin matches size of loadRect: (150vw 300vh) * 1.25.
-          {root, rootMargin: '250% 31.25%'}
-        );
+      this.intersectionObserver_ = new IntersectionObserver(
```

Comment:
Let's not remove this try/catch yet. We had another case where native was throwing due to the `rootMargin`. I think we should leave this try/catch in place until we cleanup the experiment entirely.

Reply:
I didn't know about that! Ok, I'll leave the try/catch. I just rebased and force pushed!